lib/ignored-paths.js

```diff
@@ -146,7 +95,7 @@ function IgnoredPaths(options) {
     }
 
     this.defaultPatterns = DEFAULT_IGNORE_PATTERNS.concat(options.patterns || []);
-    this.baseDir = ".";
+    this.baseDir = options.cwd;
 
     this.ig = {
         custom: new ignore.Ignore({
```

Comment:
It is recommended to upgrade `node-ignore` to `3.x` to fix several known issues, such as

- paths on windows
- negative ignore rules
- and many special cases of gitignore spec

#### Tips

- `new ignore.Ignore()` -> `ignore()`
- `.addIgnoreFile(filename)` -> `.add(fs.readFileSync(filename).toString())`
- `.on('warn')` -> removed
- `ignore.Ignore.options` -> removed, no default ignore rules will be added.